SessionMessagingKit/Sending & Receiving/Notifications/Models/UnregisterResponse.swift

```diff
@@ -0,0 +1,11 @@
+// Copyright © 2022 Rangeproof Pty Ltd. All rights reserved.
+
+import Foundation
+
+extension PushNotificationAPI {
+    struct UnregisterResponse: Codable {
```

Comment:
This seems to be duplicated with `RegisterResponse`.

Reply:
Ah yep, good catch - I've combined them into a single type